docs/articles/string_concatenation.md

```diff
@@ -2,10 +2,19 @@
 
 If the <xref:NCalc.ExpressionOptions.StringConcat> option specified the result will be a string concatenation. Otherwise it will try to perform an arithmetic addition if possible or if both values are strings then the result will be a string concatenation.
 
-```c#
-try
-{
-   var expression = new Expression($"5 +'1'", ExpressionOptions.StringConcat);
-}
-catch (OverflowException ex) { }
-```
\ No newline at end of file
+## Example
+
+With this option enabled, string concatenation will always happen.
+```csharp
+var expression = new Expression($"5 + '1'", ExpressionOptions.StringConcat);
+var result = expression.Evaluate();
+Debug.Assert(result); // 51
+```
+
+## Summary
+
+| Expression        | `StringConcat` Enabled  | `StringConcat` Disabled |
+|-------------------|-------------------------|-------------------------|
+| `'L' + 'eo'`      | `'Leo'`                 | `'Leo'`                 |
+| `1 + '2'`         | `'12'`                  | `3`                     |
+| `'1' + '2'`       | `'12'`                  | `12`                    |
```

Comment:
This seems to be incorrect, because `'1' + '2' without `StringConcat` it will try to perform an arithmetic addition, so, the result will be `3`